lib/inferno/dsl/runnable.rb

```diff
@@ -464,6 +464,19 @@ def required_suite_options(suite_option_requirements)
           end
       end
 
+      # Remove a child test/group
+      #
+      # @param id_to_remove (Symbol, String)
```

Comment:
We use `[` instead of `(` everywhere else in the docs, so might as well keep it consistent.